students/Zach Cooper/lesson03/basic_operations.py

```diff
@@ -0,0 +1,103 @@
+""" Basic Operations"""
+import logging
+import peewee as pw
+from customers_model import Customer
+
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+
+def add_customer(customer_id, first_name, last_name, home_address,
+                 phone_number, email_address, status, credit_limit):
+    """
+        Returns new customer with **kwargs info
+    """
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
+    new_customer = Customer.create(
+        customer_id=customer_id,
+        first_name=first_name,
+        last_name=last_name,
+        home_address=home_address,
+        phone_number=phone_number,
+        email_address=email_address,
+        status=status,
+        credit_limit=credit_limit)
+    new_customer.save()
+
+    LOGGER.info('Customer database add successful')
+    LOGGER.info('%s %s has been created', Customer.first_name,
+                Customer.last_name)
+
+
+def search_customer(customer_id):
+    """
+        Search for customer name in database
+        :return: Customer info if Ture, empty dict if None
+        :param customer_id: customer_id to search for
+    """
+    find_customer_dict= {}
+    LOGGER.info("Search for customer name in database...")
+    try:
+        find_customer = Customer.get(Customer.customer_id == customer_id)
+        if find_customer:
+
+            find_customer_dict = {"first_name": find_customer.first_name,
+                                  "last_name": find_customer.last_name,
+                                  "email_address": find_customer.email_address,
+                                  "phone_number": find_customer.phone_number}
+
+        LOGGER.info('%s %s is in the database. There phone # is %s',
+                    find_customer.first_name, find_customer.last_name,
+                    find_customer.phone_number)
+
+    except pw.DoesNotExist:
+        LOGGER.info('No customer with customer id of %s', Customer.customer_id)
+
+        return find_customer_dict
```

Comment:
I don't think you want your return statement in your except block. This means it is only returned if it is empty. You also have too much in your try block. The only line that will give you a pw.DoesNotExist is the Customer.get line, so that should be the only line in your try. Try has an else block for code that you only want to run when the try succeeds, and then you can return find_customer in all cases. Note that if you have an else block, you don't need the conditional if find_customer, since it will only enter the else if the get succeeds.